<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Запомните произведение</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="element_name_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
.QLineEdit {
  background-color: #fafafa;
  border:0;
  box-shadow:0 0 4px rgba(0,0,0,0.3);
  transition: .3s box-shadow;
}
.QLineEdit:hover {
  box-shadow:0 0 4px rgba(0,0,0,0.5);
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>81</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="textcolor_label">
      <property name="styleSheet">
       <string notr="true">background-color: black
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="backcolor_label">
      <property name="styleSheet">
       <string notr="true">background-color: white
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="bordercolor_label">
      <property name="styleSheet">
       <string notr="true">background-color: grey
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="textcolor_btn">
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff;
  border-radius: 5px;
  padding: 10px 25px;
  font-family: 'Lato', sans-serif;
  font-weight: 500;
  background-color: #6495ED;</string>
      </property>
      <property name="text">
       <string>Выберите цвет текста</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backcolor_btn">
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff;
  border-radius: 5px;
  padding: 10px 25px;
  font-family: 'Lato', sans-serif;
  font-weight: 500;
  background-color: #6495ED;</string>
      </property>
      <property name="text">
       <string>Выберите цвет фона</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bordercolor_btn">
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff;
  border-radius: 5px;
  padding: 10px 25px;
  font-family: 'Lato', sans-serif;
  font-weight: 500;
  background-color: #6495ED;</string>
      </property>
      <property name="text">
       <string>Выберите цвет рамки</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="description_edit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>271</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
.QPlainTextEdit {
  background-color: #fafafa;
  border:0;
  box-shadow:0 0 4px rgba(0,0,0,0.3);
  transition: .3s box-shadow;
}
.QPlainTextEdit:hover {
  box-shadow:0 0 4px rgba(0,0,0,0.5);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;
  border-radius: 5px;
  padding: 10px 25px;
  font-family: 'Lato', sans-serif;
  font-weight: 500;
  background-color: #4dccc6;
  background-image: linear-gradient(315deg, #4dccc6 0%, #96e4df 74%);
  line-height: 42px;
  padding: 0;</string>
   </property>
   <property name="text">
    <string>Создать элемент</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
  width: 130px;
  height: 40px;
  color: #fff;
  border-radius: 5px;
  padding: 10px 25px;
  font-family: 'Lato', sans-serif;
  font-weight: 500;
  background: transparent;
  cursor: pointer;
  position: relative;
  display: inline-block;
  outline: none;
}
.QPushButton {
  width: 130px;
  height: 40px;
  line-height: 42px;
  padding: 0;
  border: none;
  background: rgb(255,27,0);
}
.btn-5:hover {
  color: #f0094a;
  background: transparent;
   box-shadow:none;
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
